cmake/Platform/Windows/PackagingPostBuild.cmake

```diff
@@ -32,6 +32,9 @@ set(_addtional_defines
     -dCPACK_RESOURCE_PATH=${CPACK_SOURCE_DIR}/Platform/Windows/Packaging
 )
 
+file(REAL_PATH "${CPACK_SOURCE_DIR}/.." _root_path)
+file(TO_NATIVE_PATH "${_root_path}/scripts/signer/signer.ps1" _sign_script)
```

Comment:
That won't work unfortunately.  Only `CPACK_` and select `CMAKE_` variables are available during cpack execution.

Reply:
ah, gotcha, this gets called from the cpack script... we can inject the definition of that variable if needed
My concern is that `CPACK_SOURCE_DIR` may not necessarily be one level inside `LY_ROOT_FOLDER`

Reply:
Thanks, wasn't sure if there was a better way to get that.

Reply:
`CPACK_SOURCE_DIR` (and `CPACK_BINARY_DIR`) are non-standard variables which get cached by cpack during cmake configure because of the "CPACK_" prefix.  They are defined here:
https://github.com/o3de/o3de/blob/bee4346df832f425358ca1ad0f3c7ab95a3f39b7/cmake/Packaging.cmake#L60-L62
So there is a high level of confidence it will point to one level below root

Reply:
ah, thanks, didnt realize we are defining that
dropping it